runtime/src/accounts_db.rs

```diff
@@ -130,9 +130,8 @@ impl AccountStorage {
         slot: Slot,
         store_id: AppendVecId,
     ) -> Option<Arc<AccountStorageEntry>> {
-        self.0
-            .get(&slot)
-            .and_then(|storage_map| storage_map.value().read().unwrap().get(&store_id).cloned())
+        self.get_slot_stores(slot)
```

Comment:
maybe is this proactive measure for the unrelated possible slow down?

Reply:
yeah this was just something else I suspected while investigating the root cause